src/warnings/features/_links.scss

```diff
@@ -4,3 +4,12 @@ a[target = "_blank"] {
     @include contentMessage(warning, "W0006", "Links that open in a new tab or window should be communicated to the user, ensure the user knows of this behavior.");
   }
 }
+
+/* W0010: Anchor tags should not be used as buttons. Links should redirect to a resource/page, if they don't they probably should be buttons. */
+a[href = "#"],

```

Comment:
Nice CSS! Maybe could do with a minor refactor:

```scss
a {
  &[href = "#"],
  &[role = "button"],
  &[href ^= "javascript:"] {
    &::after {
      @include contentMessage(warning, "W0010", "Anchor tags should not be used as buttons. Links should redirect to a resource/page, if they don't they probably should be buttons.");
    }
  }
}
```

Reply:
Aha thats nice, thanks for showing, didn't knew that